src/components/DistanceRequest.js

```diff
@@ -80,6 +80,36 @@ function DistanceRequest({transactionID, transaction, mapboxAccessToken}) {
     const numberOfWaypoints = _.size(waypoints);
     const lastWaypointIndex = numberOfWaypoints - 1;
 
+    const waypointsData = _.filter(
```

Comment:
NAB: Let's use a more descriptive name since this is now an object of `waypointMarkers`.

Reply:
changed 👍